session/bootstrap.go

```diff
@@ -1593,7 +1596,7 @@ func upgradeToVer64(s Session, ver int64) {
 	if ver >= version64 {
 		return
 	}
-	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN `Repl_slave_priv` ENUM('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Execute_priv`", infoschema.ErrColumnExists)
+	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN `Repl_slave_priv` ENUM('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Create_Tablespace_Priv`", infoschema.ErrColumnExists)
```

Comment:
I think it's better to follow MySQL. Like
```mysql
show create table mysql.user;
...
  `Super_priv` enum('N','Y') CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'N',
  `Create_tmp_table_priv` enum('N','Y') CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'N',
  `Lock_tables_priv` enum('N','Y') CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'N',
  `Execute_priv` enum('N','Y') CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'N',
  `Repl_slave_priv` enum('N','Y') CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'N',
  `Repl_client_priv` enum('N','Y') CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'N',
  `Create_view_priv` enum('N','Y') CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'N',
  `Show_view_priv` enum('N','Y') CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'N',
  `Create_routine_priv` enum('N','Y') CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'N',
...
```

We can change the create stmt. https://github.com/pingcap/tidb/blob/master/session/bootstrap.go#L92

Reply:
I think this a good idea, but the will make now new cluster mysql.user columns orders which is different with previous cluster columns orders, does that cause problem?

Reply:
I don't think it will be a big problem. But don't forget change `insert` stmt in `doDMLWorks`.

Reply:
done, I hope you can check the latest changes,  thank you.

Reply:
Also need an `upgradeToVerXX` to change the user table which create by old version like 87.

Reply:
done.

Reply:
Please also update `doDMLWorks` func to fix the UT